### ISSUE TITLE
Can't delete ephemeral replies on djs v13

### ISSUE THREAD

**Issue by Raraph84** [discordjs/discord.js#8810]
### Which package is this bug report for?

discord.js

### Issue description

I have an error when I try to delete ephemeral with djs v13.
This issue is fixed with djs v14 with [this commit](https://github.com/discordjs/discord.js/commit/fc107744618857bf28c2167f204253baf690ede8) that wasn't backported to djs v13.
Normally we can delete ephemeral replies from [this Discord API changelog](https://discord.com/developers/docs/change-log#delete-ephemeral-messages)
Step to reproduce : 
1. Reply to an interaction with an ephemeral response
2. Try deleting this response with CommandInteraction#deleteReply
3. Get Error [INTERACTION_EPHEMERAL_REPLIED]: Ephemeral responses cannot be deleted.

### Code sample

```typescript
client.on("interactionCreate", async (interaction) => {
    await interaction.reply({ content: "Replied!", ephemeral: true });
    await interaction.deleteReply();
});
```

### Package version

13.12.0

### Node.js version

v18.11.0

### Operating system

Debian 11 (Bullseye)

### Priority this issue should have

Medium (should be fixed soon)

### Which partials do you have configured?

No Partials

### Which gateway intents are you subscribing to?

Guilds, GuildMembers, GuildInvites, GuildPresences, GuildMessages, GuildMessageReactions

**Comment by Raraph84** [comment]
I made a PR [here](https://github.com/discordjs/discord.js/pull/8811)
It's my first PR on discord.js so I don't know if I did it right

**Comment by CitTheDev** [comment]
This issue should now be closed as PR #8811 has been merged
